src/components/OrderPanel/BookingTimeForm/FieldDateAndTimeInput.js

```diff
@@ -521,38 +693,9 @@ const FieldDateAndTimeInput = props => {
     }
   }, [currentMonth, currentMonthInProgress, nextMonthInProgress, monthlyTimeSlots, timeZone]);
 
-  const classes = classNames(rootClassName || css.root, className);
-
-  const bookingStartDate = values.bookingStartDate?.date || null;
-  const bookingStartTime = values.bookingStartTime || null;
-  const bookingEndDate = values.bookingEndDate?.date || null;
-
-  // Currently available monthly data
-  const [startMonth, endMonth] = getMonthlyFetchRange(monthlyTimeSlots, timeZone);
-  const timeSlotsData = timeSlotsPerDate(startMonth, endMonth, allTimeSlots, timeZone);
-  const bookingStartIdString = stringifyDateToISO8601(bookingStartDate, timeZone);
-  const timeSlotsOnSelectedDate = timeSlotsData[bookingStartIdString]?.timeSlots || [];
-
-  const availableStartTimes = getAvailableStartTimes(
-    intl,
-    timeZone,
-    bookingStartDate,
-    timeSlotsOnSelectedDate
-  );
-
-  const firstAvailableStartTime =
-    availableStartTimes.length > 0 && availableStartTimes[0] && availableStartTimes[0].timestamp
-      ? availableStartTimes[0].timestamp
-      : null;
-
-  const { startTime, endDate, selectedTimeSlot } = getAllTimeValues(
-    intl,
-    timeZone,
-    timeSlotsOnSelectedDate,
-    bookingStartDate,
-    bookingStartTime || firstAvailableStartTime,
-    bookingEndDate || bookingStartDate
-  );
+  useEffect(() => {
+    setSeatsOptions(seatsOptions);
```

Comment:
When opening /styleguide page, this component throws an uncaught type error

```
FieldDateAndTimeInput.js:697 Uncaught TypeError: setSeatsOptions is not a function
    at FieldDateAndTimeInput._selectedTimeSlot$att2 (FieldDateAndTimeInput.js:697:1)
```

I'm not yet sure if this needs to be fixed in BookingTimeForm.example.com or FieldDateAndTimeInput.example.com

Reply:
@SariSaar I noticed this too. I pushed a fix.